tools/ARIAtools/product.py

```diff
@@ -811,7 +976,7 @@ def __run__(self):
             LOGGER.debug([
```

Comment:
logger should be fed basic strings or something to be lazy formatted and not lists
last line not aligned correctly

Reply:
This part seemed to be inherited in the original refactoring effort (i.e. I didn't touch it). I will adjust as appropriate though.